examples/c_app.py

```diff
@@ -21,6 +24,8 @@ def __init__(self, loop, my_id, server, port, timeout, verbose):
         led = Pin(2, Pin.OUT, value=1)  # Optional LED
         self.cl = client.Client(loop, my_id, server, port, timeout, self.constate, None, verbose, led)
         loop.create_task(self.start(loop))
+        self.latency_added = 0
+        self.count = 0
```

Comment:
Just changed the example to measure the average latency of sending qos messages and send messages as quickly as possible without sleeping.
Also changed for header support..